services/prometheus/prometheus.go

```diff
@@ -264,32 +268,56 @@ func (svc *Service) addScrapeConfigs(cfg *config.Config, q *reform.Querier, s *m
 	return nil
 }
 
+func (svc *Service) baseConfig(filename string, s models.MetricsResolutions) *config.Config {
+	cfg := &config.Config{
+		GlobalConfig: config.GlobalConfig{
+			// these values will be overwriten if they exit in the base config gfile
+			ScrapeInterval:     model.Duration(s.LR),
+			ScrapeTimeout:      scrapeTimeout(s.LR),
+			EvaluationInterval: model.Duration(s.LR),
+		},
+	}
+
+	buf, err := ioutil.ReadFile(filename)
+	if err != nil {
+		return cfg
```

Comment:
We should log errors other than `os.IsNotExist` (i.g. file exists, but can't be read due to permissions, etc.)